media-libs/avidemux-plugins/avidemux-plugins-9999.ebuild

```diff
@@ -63,14 +65,23 @@ RDEPEND="
 	xvid? ( media-libs/xvid:0 )
 	vorbis? ( media-libs/libvorbis:0 )
 	vpx? ( media-libs/libvpx:0 )
+	system-a52dec? ( media-libs/a52dec:0 )
+	system-libass? ( media-libs/libass:0= )
+	system-libmad? ( media-libs/libmad:0 )
+	system-libmp4v2? ( media-libs/libmp4v2:0 )
 	${PYTHON_DEPS}
 "
 DEPEND="${RDEPEND}
 	oss? ( virtual/os-headers:0 )"
 
 S="${WORKDIR}/${MY_P}"
 
-PATCHES=( "${FILESDIR}"/${PN}-2.6.4-optional-pulse.patch )
+PATCHES=( "${FILESDIR}"/${PN}-2.6.20-optional-pulse.patch )
+
+src_prepare() {
+	# Don't call cmake-utils_src_prepare since there is no CMakeLists.txt in ${S}
```

Comment:
> keep patches + call default here I think

> I think it would still be useful to call it for the subdirs where CMakeLists.txt is present; just use a for loop

Keeping `PATCHES` and calling `cmake-utils_src_prepare` for the subdirs will end up trying to apply the patch multiple times in mutiple subdirs and fail.